frontend/modelarchive/src/main/java/com/amazonaws/ml/mms/archive/Manifest.java

```diff
@@ -190,20 +190,19 @@ public void setModelVersion(String modelVersion) {
             this.modelVersion = modelVersion;
         }
 
-        public String getParametersFile() {
-            return parametersFile;
+        public Map<String, Object> getExtensions() {
+            return extensions;
         }
 
-        public void setParametersFile(String parametersFile) {
-            this.parametersFile = parametersFile;
+        public void setExtensions(Map<String, Object> extensions) {
+            this.extensions = extensions;
         }
 
-        public String getSymbolFile() {
-            return symbolFile;
-        }
-
-        public void setSymbolFile(String symbolFile) {
-            this.symbolFile = symbolFile;
+        public void addExtension(String key, Object value) {
+            if (extensions == null) {
+                extensions = new LinkedHashMap<>();
+            }
+            extensions.put(key, value);
```

Comment:
Doesn't this rewrite a key's value, if the key already exists? Should we throw warning in case where users are unknowingly creating duplicate key-value extensions?

Reply:
This is a setter like method, it doesn't make sense to check duplicate here.
How can you throw a warning here?
How do you know if user want to override the value of not?